graphistry/PlotterBase.py

```diff
@@ -199,7 +199,7 @@ def __init__(self, *args, **kwargs):
         # KG embeddings
         self._relation : Optional[str] = None
         self._use_feat: bool = False
-        self.triplets: Optional[List] = None # actually torch.Tensor too
+        self.triplets: Optional[List] = None 
```

Comment:
`Optional[Union[List, torch.Tensor]]`

Reply:
No best to leave it out or would need to import torch...

Reply:
Maybe:

```
from graphistry.torch import TT
```

```
if TYPECHECKING:
  TT = 
else:
  TT =
```